### PR TITLE
A/P failure testing #495

### PR DESCRIPTION
**Pull request by ryanemerson**

### CODE REVIEW

**Comment by kami619** [comment]
@ryanemerson there seems to be a failure in the Ascii docs tests

**Review by kami619 — APPROVED** [review]
overall the changes look good to me. We can work any fixes if need be in subsequent issues and tickets.

File: ansible/roles/benchmark/tasks/install.yml [hunk 1]
```diff
@@ -14,6 +14,16 @@
 - name: Install jq
   package: name=jq state=present
 
+- name: Install dig
+  package: name=bind-utils state=present
+
+- name: Download & install OpenShift client
```
Comment (pruivo) [inline ansible/roles/benchmark/tasks/install.yml:20]: I don't think you are using `oc` anywhere.

**Review by pruivo — APPROVED** [review]

**Comment by ryanemerson** [comment]
Docs should be fixed. I just needed to escape `${DOMAIN}` with `$\{DOMAIN}`.

File: ansible/roles/benchmark/tasks/install.yml (hunk 1 above)
Comment (ryanemerson) [reply ansible/roles/benchmark/tasks/install.yml:20]: We need `oc` to login to the Openshift cluster before we execute the `kc-failover.sh` script.

**Review by mhajas — APPROVED** [review]
Thanks @ryanemerson

**Pull request merged**